src/js/components/Select/index.d.ts

```diff
@@ -51,8 +64,9 @@ export interface SelectProps {
   valueKey?:
     | string
     | { key: string; reduce?: boolean }
-    | ((...args: any[]) => any);
+    | ((...args: any[]) => string);
```

Comment:
Could this also return a number?

Reply:
hmm, from I've seen in the component source code, it depends on the value prop,
So I think it's better to leave as any